run_bringup.bash

```diff
@@ -0,0 +1,11 @@
+docker rm ds4_drv_container
```

Comment:
Please add a `#!/bin/bash` at the beginning of the file.

Reply:
Resolved